bftengine/include/bftengine/Replica.hpp

```diff
@@ -78,6 +79,12 @@ class IReplica {
   using IReplicaPtr = std::unique_ptr<IReplica>;
   static IReplicaPtr createNewReplica(
       ReplicaConfig *, IRequestsHandler *, IStateTransfer *, bft::communication::ICommunication *, MetadataStorage *);
+  static IReplicaPtr createNewReplica(ReplicaConfig *,
+                                      IRequestsHandler *,
+                                      IStateTransfer *,
+                                      bft::communication::ICommunication *,
+                                      MetadataStorage *,
+                                      bool &);
```

Comment:
Can you please give a name to this bool? Better yet make it a concrete type (enum). Overloading a bool parameter makes calling code difficult to understand.

Reply:
I would prefer to keep using bool due to code simplicity

Reply:
No problem.